__tests__/app.test.js

```diff
@@ -0,0 +1,31 @@
+const app = require("../app");
+const request = require("supertest");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const {articleData, commentData, topicData, userData } = require('../db/data/test-data/index')
+
+beforeEach(() => seed({ topicData, userData, articleData, commentData }));
+afterAll(() => db.end());
+
+describe('/api/topics',()=>{
+    test('200: response array should contains data for all topics',()=>{
+        return request(app)
+        .get('/api/topics')
+        .expect(200)
+        .then(({body: {topics}}) =>{
+            expect(topics.length).toBe(3)
+            topics.forEach(({description, slug})=>{
+                expect(typeof description).toBe('string')
```

Comment:
Be careful with checking the type of description to be a string as it is possible for it to be null (if you check the seed file)